sklearn_bench/df_clsf.py

```diff
@@ -46,7 +43,8 @@
 
 params = bench.parse_args(parser)
 
-from sklearn.ensemble import RandomForestClassifier
+if not params.no_intel_optimized:
+    from sklearn.ensemble import RandomForestClassifier
```

Comment:
Where is RandomForestClassifier if params.no_intel_optimized=True? Is benchmark predicted to be failed in this case?

Reply:
My bad. I just wanted to separate this import from others, so we can ISort them

Reply:
changed the behaviour. It should work good now